packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/ModuleRegistry.kt

```diff
@@ -63,14 +64,19 @@ class ModuleRegistry(
 
   fun <T : View> getModuleHolder(viewClass: Class<T>): ModuleHolder<*>? {
     return registry.firstNotNullOfOrNull { (_, holder) ->
-      if (holder.definition.viewManagerDefinition?.viewType == viewClass) {
+      val definition = getViewDefinition(holder, viewClass)
+      if (definition != null) {
         holder
       } else {
         null
       }
```

Comment:
```suggestion
      holder.takeIf { getViewDefinition(holder, viewClass) != null }
```
[`takeIf`](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin/take-if.html) is Kotlin build in function.